doc/source/cli_accuracy.md

```diff
@@ -0,0 +1,384 @@
+            Its use is entirely the responsibility of the user.
+
+      .. code-block:: yaml
+
+        coregistration:
+          step_one:
+            method: "NuthKaab"
+            extra_informations : {"max_iterations": 10}
+          step_two:
+            method: "DHMinimize"
+          sampling_grid: "reference_elev"
+          process: True
+
+      other example :
+
+      .. code-block:: yaml
+
+        coregistration:
+          step_one:
+            method: "VerticalShift"
+          sampling_grid: "reference_elev"
+          process: True
+
+   .. tab:: statistics
+
+      **Required:** No
+
+      Statistics step information. This section relates to the computed statistics:
+        1. If no block is specified, all available statistics are calculated by default.
+        [mean, median, max, min, sum, sum of squares, 90th percentile, LE90, nmad, rmse, std, valid count, total count,
+        percentage valid points, inter quartile range]
+        2. If a block is specified but no statistics are provided, then no statistics will be computed.
+
+      .. code-block:: yaml
+
+         statistics:
+
+        3. If a block is specified and some statistics are provided, then only these statistics are computed.
+
+      .. code-block:: yaml
+
+         statistics:
+           - min
+           - max
+           - mean
+
+      If a mask is provided, the statistics are also computed inside the mask.
+
+   .. tab:: outputs
+
+     **Required:** No
+
+     Outputs information.
+     Operates by levels:
+     1. Level 1 → aligned elevation only
+     2. Level 2 → more detailed output
+
+     .. csv-table:: Outputs parameters
+       :header: "Name", "Description", "Type", "Default value", "Available Value", "Required"
+       :widths: 20, 40, 10, 10, 10, 10
+
+       "path", "Path for outputs", "str", "outputs", "", "No"
+       "level", "Level for detailed outputs", "int", "1", "1 or 2", "No"
+       "output_grid", "Grid for outputs resampling", "str", "reference_elev", "reference_elev or to_be_aligned_elev", "No"
+
+     .. code-block:: yaml
+
+       outputs:
+           level : 1
+           path : "path_to/outputs"
+           output_grid: "reference_elev"
+
+     Tree of outputs for level 1 (including coregistration step)
+
+     .. code-block:: text
+
+      - root
+        ├─ tables
+        │   └─ aligned_elev_stats.csv
+        ├─ plots
+        │   ├─ diff_elev_after_coreg_map.png
+        │   ├─ diff_elev_before_coreg_map.png
+        │   ├─ diff_elev_before_after_hist.png
+        │   ├─ reference_elev_map.png
+        │   └─ to_be_aligned_elev_map.png
+        ├─ rasters
+        │   └─ aligned_elev.tif
+        ├─ report.html
+        ├─ report.pdf
+        └─ used_config.yaml
+
+     Tree of outputs for level 2 (including coregistration step)
+
+     .. code-block:: text
+
+      - root
+        ├─ tables
+        │   ├─ aligned_elev_stats.csv
+        │   ├─ diff_elev_after_coreg_stats.csv
```

Comment:
I would include this and next in the level 1 output, as they are always useful to assess the coregistration performance